src/main/appOne.java

```diff
@@ -0,0 +1,11 @@
+class Test{
```

Comment:
line no: 3 - Consider using .equals() method for comparing strings instead of == operator.
line no: 10 - Method name should be more descriptive to indicate its purpose.